Sofa/Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/TriangleFEMForceField.inl

```diff
@@ -291,7 +291,13 @@ void TriangleFEMForceField<DataTypes>::accumulateForceSmall(VecCoord& f, const V
 
         // displacements
         Displacement Depl(type::NOINIT);
-        m_triangleUtils.computeDisplacementSmall(Depl, _rotatedInitialElements[elementIndex], deforme_b, deforme_c);
+        Depl[0] = 0;
+        Depl[1] = 0;
+        Depl[2] = (_initialPoints.getValue()[b][0]-_initialPoints.getValue()[a][0]) - deforme_b[0];
+        Depl[3] = 0;
+        Depl[4] = (_initialPoints.getValue()[c][0]-_initialPoints.getValue()[a][0]) - deforme_c[0];
+        Depl[5] = (_initialPoints.getValue()[c][1]-_initialPoints.getValue()[a][1]) - deforme_c[1];
+
```

Comment:
```suggestion
        auto initialPoints = _initialPoints.getValue();
        Depl[0] = 0;
        Depl[1] = 0;
        Depl[2] = (initialPoints[b][0]-initialPoints[a][0]) - deforme_b[0];
        Depl[3] = 0;
        Depl[4] = (initialPoints[c][0]-initialPoints[a][0]) - deforme_c[0];
        Depl[5] = (initialPoints[c][1]-initialPoints[a][1]) - deforme_c[1];

```
Because getValue imply to scan if the DDNode graph needs to be updated. So there is a lot of overhead thus it makes sens ot factorize.